features/events/traps/src/main/resources/META-INF/opennms/applicationContext-trapDaemon.xml

```diff
@@ -23,6 +23,19 @@
     <property name="eventReader" ref="eventReader"/>
   </bean>
   
+  <bean id="trapNotificationHandler" class="org.opennms.netmgt.trapd.TrapNotificationHandler"/>
```

Comment:
This is an interface. You cannot create a new one here. The handler that you should be using is the TrapNotificationHandlerDefaultImpl bean that is defined below.